### PR TITLE
scheduleEmberAction() needs an Ember.run when auto-run is off

### PR DESCRIPTION
**Pull request by jayphelps**

### CODE REVIEW

File: ember-cli-rx/addon/schedulers/ember-scheduler.js [hunk 1]
```diff
@@ -28,10 +28,12 @@ function scheduleAbsolute(state, dueTime, action) {
 }
 
 function scheduleEmberAction(disposable, queue, target, state, action, scheduler) {
-	Ember.run.schedule(queue, target, function() {
-		if(!disposable.isDisposed) {
-			disposable.setDisposable(action(scheduler, state));
-		}
+	Ember.run(function() {
```
Comment (machty) [inline ember-cli-rx/addon/schedulers/ember-scheduler.js]: I think `Ember.run.join` would make more sense here?

**Comment by machty** [comment]
Also I think this needs to be in https://github.com/blesh/ember-cli-rx ... pretty sure this repo just treats ember-cli-rx as an upstream.

**Comment by jayphelps** [comment]
@machty yep, you're right. Dammit @blesh!! :imp:

File: ember-cli-rx/addon/schedulers/ember-scheduler.js (hunk 1 above)
Comment (stefanpenner) [inline ember-cli-rx/addon/schedulers/ember-scheduler.js]: i believe: https://github.com/blesh/RxEmber/blob/b5f66a4ce1249d385a4b2d0f7d5b0e3adb6f6a88/ember-cli-rx/addon/schedulers/ember-scheduler.js#L20 needs the run-loop

Comment (stefanpenner) [inline ember-cli-rx/addon/schedulers/ember-scheduler.js]: even better it should use Ember.run.later instead of the timeout

Comment (stefanpenner) [inline ember-cli-rx/addon/schedulers/ember-scheduler.js]: but am unclear what invokes: 

https://github.com/blesh/RxEmber/blob/b5f66a4ce1249d385a4b2d0f7d5b0e3adb6f6a88/ember-cli-rx/addon/schedulers/ember-scheduler.js#L9

Comment (jayphelps) [inline ember-cli-rx/addon/schedulers/ember-scheduler.js]: @blesh get in here

**Comment by blesh** [comment]
LMAO this thread, tho.

**Comment by blesh** [comment]
@stefanpenner ... The run loop thing was a hack we needed to do to get tests working for @jayphelps. I thought the Ember.run.schedule should start the loop, but apparently not?

**Pull request closed without merging**